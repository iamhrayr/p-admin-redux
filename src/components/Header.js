import React, { useState } from "react";
import { Layout, Menu, Breadcrumb, Icon } from "antd";

export default () => {
  const [collapsed, setCollapse] = useState(false);

  return (
    <Layout.Header style={{ background: "#fff", padding: 0 }}>
      {/* <Icon
        className="trigger"
        type={collapsed ? "menu-unfold" : "menu-fold"}
        // onClick={this.toggle}
      /> */}
    </Layout.Header>
  );
};
