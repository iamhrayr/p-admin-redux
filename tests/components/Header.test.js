import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

describe('<Header /> component', () => {
  it('exists', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper.exists()).toBe(true);
  });

  it('match snapshot', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
