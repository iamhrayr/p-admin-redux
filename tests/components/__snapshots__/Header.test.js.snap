// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> component match snapshot 1`] = `ShallowWrapper {}`;
